patches/ios-chrome-browser-ui-webui-about_ui.cc.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/ios/chrome/browser/ui/webui/about_ui.cc b/ios/chrome/browser/ui/webui/about_ui.cc
+index 40a3563375034..b3e69be1667d1 100644
+--- a/ios/chrome/browser/ui/webui/about_ui.cc
++++ b/ios/chrome/browser/ui/webui/about_ui.cc
+@@ -98,7 +98,7 @@ std::string ChromeURLs() {
+             *i + "</a></li>\n";
+   html += "</ul>\n";
+   AppendFooter(&html);
+-  return html;
++  return brave::AboutUIChromeURLs(html);
```

Comment:
`#define kChromeUIChromeURLsHost kChromeUIChromeURLsHost) { response = brave::AboutUIChromeURLs(ChromeURLs()); } else if (false` ?

Reply:
This will give an error:
```
In file included from about_ui.cc:24:
chrome_url_constants.h:35:19: error: expected ';' after top level declarator
   35 | extern const char kChromeUIChromeURLsHost[];
```

`about.cc` includes `chrome_url_constants.h` which defined `kChromeUIChromeURLsHost`. So when we do an override on that with the one suggested above, it will give a compilation error.

Reply:
> This will give an error:
> 
> ```
> In file included from about_ui.cc:24:
> chrome_url_constants.h:35:19: error: expected ';' after top level declarator
>    35 | extern const char kChromeUIChromeURLsHost[];
> ```
> 
> `about.cc` includes `chrome_url_constants.h` which defined `kChromeUIChromeURLsHost`. So when we do an override on that with the one suggested above, it will give a compilation error.

did you include `chrome_url_constants.h` before adding `#define kChromeUIChromeURLsHost`?

Reply:
Fixed in: https://github.com/brave/brave-core/pull/20929/commits/28844bde3166f309f9a3e7c9b72c59646c65d4fa